todo/src/components/Task.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { toggleComplete } from '../actions';
+import { connect } from 'react-redux';
+
+class Task extends Component {
+
+  toggleComplete(event) {
```

Comment:
Where are you calling this function? I think you could keep this component a presentational component and pass toggleComplete down through props from App.js. What do you think? Then you wouldn't need the connect function.